checks/evaluation/pinned_dependencies.go

```diff
@@ -224,15 +113,15 @@ func PinningDependencies(name string, c *checker.CheckRequest,
 				lm.Remediation = probeRemToRuleRem(f.Remediation)
 			}
 			dl.Warn(lm)
-		case finding.OutcomeNotAvailable:
+		case finding.OutcomeError:
```

Comment:
should probably have a test for this. either now or in followups.

Reply:
> should probably have a test for this. either now or in followups.

If possible, let's do it in a follow-up.